### ISSUE TITLE
Variable product prices don't get updated at end of the scheduled time

### ISSUE THREAD

**Issue by ttarpinyan** [woocommerce/woocommerce#12004]
## EXPLANATION OF THE ISSUE

WC Version: 2.6.4
WP Version: 4.6.1
When we schedule a sales event on variable products, at the end of the sale period "Sale price", "Sale start date", "Sale end date" fields on the wp-admin disappear as expected. However, on the store front  the product price does not revert back to the regular price.

## STEPS TO REPRODUCE THE ISSUE

Pick any product with variations
Enter a sale price
Set the "Sale start date"
Set the "Sale end date"
Wait until the cron job kick in end clear all the fields above in the wp-admin
Check the front store front. You'll see that the price is not reverted back to the regular price unless you manually hit "Update" on the wp-admin product detail page.

## SYSTEM STATUS REPORT

```
`
### WordPress Environment ###

Home URL: http://bigchill.com
Site URL: http://bigchill.com
WC Version: 2.6.4
Log Directory Writable: ✔
WP Version: 4.6.1
WP Multisite: –
WP Memory Limit: 256 MB
WP Debug Mode: –
WP Cron: ✔
Language: en_US

### Server Environment ###

Server Info: nginx/1.10.0
PHP Version: 5.6.26
PHP Post Max Size: 100 MB
PHP Time Limit: 120
PHP Max Input Vars: 1000
cURL Version: 7.19.7
NSS/3.21 Basic ECC

SUHOSIN Installed: –
MySQL Version: 5.6.32
Max Upload Size: 100 MB
Default Timezone is UTC: ✔
fsockopen/cURL: ✔
SoapClient: ✔
DOMDocument: ✔
GZip: ✔
Multibyte String: ✔
Remote Post: ✔
Remote Get: ✔

### Database ###

WC Database Version: 2.6.4
: 
woocommerce_sessions: ✔
woocommerce_api_keys: ✔
woocommerce_attribute_taxonomies: ✔
woocommerce_downloadable_product_permissions: ✔
woocommerce_order_items: ✔
woocommerce_order_itemmeta: ✔
woocommerce_tax_rates: ✔
woocommerce_tax_rate_locations: ✔
woocommerce_shipping_zones: ✔
woocommerce_shipping_zone_locations: ✔
woocommerce_shipping_zone_methods: ✔
woocommerce_payment_tokens: ✔
woocommerce_payment_tokenmeta: ✔

### Active Plugins (20) ###

Gravity Forms: by rocketgenius – 2.0.7
Advanced Custom Fields Repeater Collapser: by Mark Root-Wiley – 1.4.3
Advanced Custom Fields PRO: by Elliot Condon – 5.4.6
Better Search Replace: by Delicious Brains – 1.2.10
Disqus Comment System: by Disqus  – 2.85
Enhanced E-commerce for Woocommerce store: by Tatvic – 1.0.18
Woocommerce Product Feed: by Victor Ciobanu and Voltage – 99.0
Gravity Forms MailChimp Add-On: by rocketgenius – 3.7.1
Regenerate Thumbnails: by Alex Mills (Viper007Bond) – 2.2.6
WooCommerce Admin Custom Order Fields: by WooThemes / SkyVerge – 1.6.3
WooCommerce Customer/Order CSV Export: by WooThemes / SkyVerge – 4.0.7
WooCommerce Affirm Gateway: by WooThemes – 1.0.3
WooCommerce Authorize.Net AIM Gateway: by WooThemes / SkyVerge – 3.9.2
WooCommerce Product Add-ons: by Automattic – 2.7.24
WooCommerce: by WooThemes – 2.6.4
WooCommerce Helper: by WooCommerce – 1.7.1
WordPress Importer: by wordpressdotorg – 0.6.3
Yoast SEO: by Team Yoast – 3.6
WP Smush: by WPMU DEV – 2.4.5
YoRocket: by Backlinko LLC – 1.5.0

### Settings ###

Force SSL: ✔
Currency: USD ($)
Currency Position: left
Thousand Separator: ,
Decimal Separator: .
Number of Decimals: 2

### API ###

API Enabled: ✔

### WC Pages ###

Shop Base: #5 - /shop-unused/
Cart: #6 - /cart/
Checkout: #7 - /checkout/
My Account: #8 - /my-account/

### Taxonomies ###

Product Types: external (external)
grouped (grouped)
simple (simple)
variable (variable)


### Theme ###

Name: Big Chill
Version: 1.0.0
Author URL: http://voltagead.com
Child Theme: ✔
Parent Theme Name: The HTML5 Reset Theme
Parent Theme Version: 2.0
Parent Theme Author URL: http://monkeydo.biz
WooCommerce Support: ✔

### Templates ###

Overrides: bigchill-child/woocommerce/archive-product.php
bigchill-child/woocommerce/cart/cart-empty.php
bigchill-child/woocommerce/cart/cart-shipping.php version 2.3.0 is out of date. The core version is 2.5.0
bigchill-child/woocommerce/cart/cart-totals.php
bigchill-child/woocommerce/cart/cart.php
bigchill-child/woocommerce/cart/mini-cart.php version 2.1.0 is out of date. The core version is 2.5.0
bigchill-child/woocommerce/cart/proceed-to-checkout-button.php
bigchill-child/woocommerce/cart/shipping-calculator.php
bigchill-child/woocommerce/checkout/form-billing.php
bigchill-child/woocommerce/checkout/form-checkout.php
bigchill-child/woocommerce/checkout/form-shipping.php
bigchill-child/woocommerce/checkout/payment.php version 2.4.7 is out of date. The core version is 2.5.0
bigchill-child/woocommerce/checkout/thankyou.php
bigchill-child/woocommerce/content-product.php version 2.4.0 is out of date. The core version is 2.6.1
bigchill-child/woocommerce/content-single-product.php
bigchill-child/woocommerce/emails/admin-new-order.php version 2.4.0 is out of date. The core version is 2.5.0
bigchill-child/woocommerce/emails/customer-completed-order.php version 2.4.0 is out of date. The core version is 2.5.0
bigchill-child/woocommerce/emails/customer-processing-order.php version 2.4.0 is out of date. The core version is 2.5.0
bigchill-child/woocommerce/emails/email-addresses.php
bigchill-child/woocommerce/emails/email-order-items.php
bigchill-child/woocommerce/global/breadcrumb.php
bigchill-child/woocommerce/global/quantity-input.php version 2.1.0 is out of date. The core version is 2.5.0
bigchill-child/woocommerce/global/sidebar.php
bigchill-child/woocommerce/global/wrapper-start.php
bigchill-child/woocommerce/notices/error.php
bigchill-child/woocommerce/notices/notice.php
bigchill-child/woocommerce/notices/success.php
bigchill-child/woocommerce/order/order-details-customer.php
bigchill-child/woocommerce/order/order-details-item.php version 2.4.0 is out of date. The core version is 2.5.0
bigchill-child/woocommerce/order/order-details.php version 2.4.0 is out of date. The core version is 2.6.0
bigchill-child/woocommerce/single-product/add-to-cart/variable.php version 2.4.0 is out of date. The core version is 2.5.0
bigchill-child/single-product.php

: Learn how to update outdated templates

### Customer/Order CSV Export ###

Order Export Format: custom
Customer Export Format: default
Automatically Export Orders: email
Export Trigger: schedule
Export Start Time: 8:00am
Export Interval: 200
Next Export: Scheduled on October 4
2016 2:40 am

Order Statuses: 
Product Categories: Warning:  implode(): Invalid arguments passed in /wp-content/plugins/woocommerce-customer-order-csv-export/includes/admin/views/html-system-status-table.php on line 107
Products: 
Automatically Export Customers: disabled

### Authorize.Net AIM ###

Environment: Production
`
```

**Comment by claudiosmweb** [comment]
Closed in favor of #12005

**Comment by dnaber-de** [comment]
I just stumbled over this issue and reviewed the PR #12005: https://github.com/woocommerce/woocommerce/pull/12005#issuecomment-282018218 You might want to have a look at my comment there.
